utils/resolve.py

```diff
@@ -124,15 +135,23 @@ def retain_gear_outputs(
 
     """
     log.info("Resolving gear outputs.")
+
+    if retain_nifti == output_nrrd:
+        log.critical(
+            "Function arguments retain_nifti and output_nrrd are exclusive. Gear config logic is broken. Exiting."
+        )
+        os.sys.exit(1)
+
     # fmt: off
```

Comment:
The purpose of a code formatter is to make it easier to read and consistent across the codebase. I make sure to double-check between the `#fmt: off` and `#fmt: on` for consistency with black. However, if you run black on this script, it will drastically change the structure and spacing, making it extremely difficult to read and debug. If you notice, there are several sections that are very similar in nature, and maintaining the consistency of structure across the script is more important for longevity and maintenance. In addition, this is the only place, less than 100 lines of code, in this entire Gear where I have elected to turn the black formatter off, so I do not believe this is an issue worth attention. As such, I will be retaining the `#fmt off/on` options in this script.